db/test/fns/queue_test.js

```diff
@@ -1159,6 +1179,78 @@ suite(testing.suiteName(), function() {
       assert.deepEqual(res[0].recent_tasks, []);
     });
 
+    helper.dbTest('queue_worker_seen creates rows', async function(db) {
+      const expires = taskcluster.fromNow('1 day');
+      await db.fns.queue_worker_seen({
+        task_queue_id_in: 'prov/wt',
+        worker_group_in: 'wg',
+        worker_id_in: 'wi',
+        expires_in: expires,
+      });
+
+      const res = await db.fns.get_queue_worker_tqid('prov/wt', 'wg', 'wi', new Date());
+      assert.equal(res[0].task_queue_id, 'prov/wt');
+      assert.equal(res[0].worker_group, 'wg');
+      assert(res[0].quarantine_until < new Date()); // defaults to in the past
+      assert.deepEqual(res[0].expires, expires);
+      helper.assertDateApproximately(res[0].first_claim, new Date());
+      assert.deepEqual(res[0].recent_tasks, []);
+    });
+
+    helper.dbTest('queue_worker_seen updates rows', async function(db) {
+      // only the expires field gets updated (#4366 would add last_active_date as well)
+      const expireses = [
```

Comment:
expireses as at the plural of expires... 🤦

Reply:
I thought you'd like that ;)